src/api/posts.ts

```diff
@@ -0,0 +1,25 @@
+import { Post } from '../types';
+
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
It is recommended to use `.env` files to store this type of variables, since we can have multiple `BASE_URL` in many environments, and we don't want to have to change it in the whole codebase nor versioning it.